csatar-plugins/csatar/components/CreateFrontendAccounts.php

```diff
@@ -0,0 +1,300 @@
+<?php namespace Csatar\Csatar\Components;
+
+use Lang;
+use Auth;
+use Mail;
+use Event;
+use Flash;
+use Input;
+use Request;
+use Redirect;
+use Validator;
+use ValidationException;
+use ApplicationException;
+use October\Rain\Auth\AuthException;
+use Cms\Classes\Page;
+use Cms\Classes\ComponentBase;
+use RainLab\User\Models\User as UserModel;
+use RainLab\User\Models\Settings as UserSettings;
+use Exception;
+use Csatar\Csatar\Models\Scout;
+/**
+ * Creates a Frontend user account for an existing Scout
+ *
+ */
+//class CreateFrontendAccounts extends ComponentBase
+class CreateFrontendAccounts extends \RainLab\User\Components\Account
+{
+    public $messages = [
+        'error' => [],
+        'success' => [],
+    ];
+
+    public $scouts;
+
+    public function componentDetails()
+    {
+        return [
+            'name' => Lang::get('csatar.csatar::lang.plugin.component.createFrontendAccounts.name'),
+            'description' => Lang::get('csatar.csatar::lang.plugin.component.createFrontendAccounts.description')
+        ];
+    }
+
+    public function defineProperties()
+    {
+        return [
+            'paramCode' => [
+                'title'       => /*Reset Code Param*/'rainlab.user::lang.reset_password.code_param',
+                'description' => /*The page URL parameter used for the reset code*/'rainlab.user::lang.reset_password.code_param_desc',
+                'type'        => 'string',
+                'default'     => 'code'
+            ],
+            'resetPage' => [
+                'title'       => /* Reset Page */'rainlab.user::lang.account.reset_page',
+                'description' => /* Select a page to use for resetting the account password */'rainlab.user::lang.account.reset_page_comment',
+                'type'        => 'dropdown',
+                'default'     => ''
+            ],
+        ];
+    }
+
+    public function prepareVars()
+    {
+        $this->page['user'] = $this->user();
+        $this->page['canRegister'] = $this->canRegister();
+        $this->page['loginAttribute'] = $this->loginAttribute();
+        $this->scouts = Scout::all();
+    }
+
+    public function onRun()
+    {
+        if ($redirect = $this->redirectForceSecure()) {
+            return $redirect;
+        }
+
+        $this->prepareVars();
+
+    }
+
+    public function getResetPageOptions()
+    {
+        return [
+                '' => '- current page -',
```

Comment:
Szerintem ez a szöveg is fordítható kellene legyen.

Reply:
Betéve.